spec.html

```diff
@@ -37360,25 +37360,13 @@ <h1>Array.prototype.filter ( _callbackfn_ [ , _thisArg_ ] )</h1>
       <emu-clause id="sec-array.prototype.find">
         <h1>Array.prototype.find ( _predicate_ [ , _thisArg_ ] )</h1>
         <emu-note>
-          <p>_predicate_ should be a function that accepts three arguments and returns a value that is coercible to a Boolean value. `find` calls _predicate_ once for each element of the array, in ascending order, until it finds one where _predicate_ returns *true*. If such an element is found, `find` immediately returns that element value. Otherwise, `find` returns *undefined*.</p>
-          <p>If a _thisArg_ parameter is provided, it will be used as the *this* value for each invocation of _predicate_. If it is not provided, *undefined* is used instead.</p>
-          <p>_predicate_ is called with three arguments: the value of the element, the index of the element, and the object being traversed.</p>
-          <p>`find` does not directly mutate the object on which it is called but the object may be mutated by the calls to _predicate_.</p>
-          <p>The range of elements processed by `find` is set before the first call to _predicate_. Elements that are appended to the array after the call to `find` begins will not be visited by _predicate_. If existing elements of the array are changed, their value as passed to _predicate_ will be the value at the time that `find` visits them; elements that are deleted after the call to `find` begins and before being visited are still visited and are either looked up from the prototype or are *undefined*.</p>
+          <p>`find` calls _predicate_ once for each element of the array, in ascending index order, until it finds one where _predicate_ returns *true*. If such an element is found, `find` immediately returns that element value. Otherwise, `find` returns *undefined*.</p>
```

Comment:
This is now incorrect; it doesn't care if it returns true, it care if it returns something _truthy_ (ie, the original verbage of "coercible to a Boolean value", altho that applies to every value so it's probably not ideal either)

Same comment for all the other method notes.

Reply:
The "returns true" wording has always been incorrect since A.p.find was introduced in ES6. I'd use the word 'truthy' if the spec used it anywhere else, but it doesn't. I figure some fudging is acceptable because this is just a Note; and even if it weren't, preambles are allowed to be a bit loose, aren't they?

(Note that ArrayFindViaPredicate's preamble is more precise: "returns a value that coerces to true".)

Reply:
I like that wording for here as well, personally.